spec/support/google_sheet_helper.rb

```diff
@@ -0,0 +1,30 @@
+module GoogleSheetHelper
+  def google_sheet_url(key:, gid:)
+    "https://docs.google.com/spreadsheets/d/#{key}/pub?gid=#{gid}&single=true&output=tsv"
+  end
+
+  def google_sheet_fixture(extra_rows=[])
+    [
+      google_sheet_row(content_base_path: "content_base_path" , link_title: "link_title"     , link_content_id: "link_content_id"           , link_type: "link_type")     ,
+      google_sheet_row(content_base_path: "/content-1/"       , link_title: "Education"      , link_content_id: "education-content-id"      , link_type: "taxons")        ,
+      google_sheet_row(content_base_path: "/content-1/"       , link_title: "Cabinet Office" , link_content_id: "cabinet-office-content-id" , link_type: "organisations") ,
+      google_sheet_row(content_base_path: "/content-2/"       , link_title: "Early Years"    , link_content_id: "early-years-content-id"    , link_type: "taxons")        ,
```

Comment:
Can you add a test case for adding two links of the same type to a single item? Pretty sure it works, but just to be safe.